SnipsPlatform/Ontology.swift

```diff
@@ -729,3 +729,60 @@ public struct AsrModelParameters {
         cParameters.endpointing?.freeUnsafeMemory()
     }
 }
+
+/// TextCapturedMessage
+public struct TextCapturedMessage {
+    public let text: String
+    public let tokens: CAsrTokenArray?
+    public let likelihood: Float
+    public let seconds: Float
+    public let siteId: String
+    public let sessionId: String?
+    
+    init(cTextCapturedMessage: CTextCapturedMessage) {
+        self.text = String(cString: cTextCapturedMessage.text)
+        self.tokens = nil
```

Comment:
I suggest where to add the `AsrToken` converter or to remove the attribute.

Reply:
Forgot this 😅 Added